src/SlackMessages.ts

```diff
@@ -1,13 +1,11 @@
-import { Presentable } from "@atomist/rug/operations/Handlers";
-
 /**
  * Construct and render slack messages according to Slack message
  * formatting: https://api.slack.com/docs/message-formatting. Customize
  * messages with rug actions.
  */
 
 export function emptyString(str: string) {
-    return !str || str === "" || str === undefined;
+    return str == null || str === "";
```

Comment:
Why does the test against `null` only have two equal signs? Does that cover both `null` and `undefined`? I've been wondering about that.